assets/src/pages/room/components/StreamPlayer/simulcast/SimulcastEncodingToReceive.tsx

```diff
@@ -3,28 +3,38 @@ import { TrackEncoding } from "@jellyfish-dev/membrane-webrtc-js";
 import { isTrackEncoding } from "../../../../types";
 
 type Props = {
-  setDesiredEncoding: (encoding: TrackEncoding) => void;
-  desiredEncoding?: TrackEncoding;
+  setTargetEncoding: (encoding: TrackEncoding | null) => void;
+  targetEncoding: TrackEncoding | null;
+  userSelectedEncoding: TrackEncoding | null;
   disabled?: boolean;
 };
 
-export const SimulcastEncodingToReceive: FC<Props> = ({ desiredEncoding, setDesiredEncoding, disabled }: Props) => {
+export const SimulcastEncodingToReceive: FC<Props> = ({
+  userSelectedEncoding,
+  targetEncoding,
+  setTargetEncoding,
+  disabled,
+}: Props) => {
   return (
     <div className="absolute bottom-0 right-0 z-50 bg-white p-2 text-sm text-gray-700 opacity-80 md:text-base">
-      <label>Encoding to receive</label>
-      <select
-        disabled={disabled}
-        value={desiredEncoding}
-        onChange={(event) => {
-          const value = event.target.value;
-          if (!isTrackEncoding(value)) return;
-          setDesiredEncoding(value);
-        }}
-      >
-        <option value="h">High</option>
-        <option value="m">Medium</option>
-        <option value="l">Low</option>
-      </select>
+      <div>Target encoding: {targetEncoding}</div>
+      <div>
+        <label>Highest encoding to receive</label>
+        <select
+          disabled={disabled}
+          value={userSelectedEncoding || undefined}
+          onChange={(event) => {
+            const value = event.target.value;
+            if (!isTrackEncoding(value) && value !== "") return;
+            setTargetEncoding(value === "" ? null : value);
+          }}
+        >
+          <option value=""></option>
```

Comment:
selecting the empty option should invoke useEffect in`useAutomaticEncodingSwitching`

Reply:
I changed that to: https://github.com/membraneframework/membrane_videoroom/pull/121#issuecomment-1431505952